Cargo.toml

```diff
@@ -57,7 +59,8 @@ indexmap = "1.6.2"
 hex = "0.4.2"
 base64 = "0.13.0"
 lazy_static = "1.4.0"
-uuid = { version = "0.8.1", features = ["serde", "v4"] }
+uuid-0_8 = { package = "uuid", version = "0.8.1", features = ["serde", "v4"], optional = true }
+uuid_1_lib = { package = "uuid", version = "1.1.2", features = ["serde", "v4"] }
```

Comment:
Features and deps are in the same namespace, even for non-optional deps, so this couldn't be called just `uuid_1`.